features/facebookImport/src/components/importExplanationExpandable/importExplanationExpandable.jsx

```diff
@@ -137,7 +149,7 @@ const ImportExplanationExpandable = ({
                 <button
                     className={`btn-highlighted ${file ? "" : "deactivated"}`}
                     onClick={
-                        file
+                        file && !fileUploadActive
                             ? () => onUpdateImportStatus(importSteps.explore)
```

Comment:
Couldn't this just be `onClick={() => file && !fileUploadActive && onUpdateImportStatus(importSteps.explore)}` ?

Or even:

```
onClick={() => {
    if (!file || fileUploadActive) return;
    onUpdateImportStatus(importSteps.explore);
}}
```

Either is shorter and less convoluted :D